.github/workflows/render.yml

```diff
@@ -0,0 +1,21 @@
+name: Render pdfs
+
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+
+  render:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Git repository
+        uses: actions/checkout@v2
+
+      - name: Compile Zcash Protocol Specification
+        uses: ./.github/actions/render-protocol-pdf
+
+      - uses: EndBug/add-and-commit@v7.5.0
+        with:
+          add: '**/*.pdf'
+          default_author: github_actions
```

Comment:
If we make the above change then this is fine as-is, but if not, then we should at least ensure that this won't make empty commits if the PDF is unaltered.

Reply:
Applied. Thanks!